### PR TITLE
[codestyle] plugins xml

### PR DESCRIPTION
**Pull request by brianteeman**
codestyle review for the xml files in the plugins folder

### CODE REVIEW

File: plugins/authentication/ldap/ldap.xml [hunk 1]
```diff
@@ -61,30 +61,30 @@
...
 
 				<field
 					name="no_referrals"
 					type="radio"
-					default="0"
-					class="btn-group btn-group-yesno"
 					label="PLG_LDAP_FIELD_REFERRALS_LABEL"
 					description="PLG_LDAP_FIELD_REFERRALS_DESC"
-				>
+					default="0"
```
Comment (Quy) [inline plugins/authentication/ldap/ldap.xml]: Swap lines `class` before `default`.

Comment (brianteeman) [inline plugins/authentication/ldap/ldap.xml]: the codestyle guide only specifies the order of name/type/label/description

**Comment by wojsmol** [comment]
I'll review later.
cc @andrepereiradasilva Plase review.

File: plugins/editors/tinymce/form/setoptions.xml [hunk 2]
```diff
@@ -1,313 +1,313 @@
...
-            class="access-select"
-            label="PLG_TINY_FIELD_SETACCESS_LABEL"
-            description="PLG_TINY_FIELD_SETACCESS_DESC"
-            labelclass="label label-success"
+        name="access"
+        type="usergrouplist"
+        multiple="true"
+        class="access-select"
+        label="PLG_TINY_FIELD_SETACCESS_LABEL"
+        description="PLG_TINY_FIELD_SETACCESS_DESC"
```
Comment (Quy) [inline plugins/editors/tinymce/form/setoptions.xml]: Move `label` and `description` below `type`.

Comment (wojsmol) [inline plugins/editors/tinymce/form/setoptions.xml]: @Quy See https://github.com/brianteeman/joomla-cms/blob/4de9fb6226d68276c7797e8fb270e28239e73c2b/plugins/editors/tinymce/form/setoptions.xml this ok

Comment (brianteeman) [inline plugins/editors/tinymce/form/setoptions.xml]: @wojsmol I fixed this earlier this morning

**Comment by Quy** [comment]
Reformat https://github.com/brianteeman/joomla-cms/blob/fd13b736c1d4e1794dc6b152b319c4eb0c4e28e1/plugins/editors/tinymce/tinymce.xml#L28

**Comment by brianteeman** [comment]
@quy did that fix

**Comment by brianteeman** [comment]
conflicts resolved

File: plugins/fields/sql/sql.xml [hunk 3]
```diff
@@ -28,6 +28,7 @@
 					description="PLG_FIELDS_SQL_PARAMS_QUERY_DESC"
 					filter="raw"
 					rows="10"
+					filter="raw"
```
Comment (zero-24) [inline plugins/fields/sql/sql.xml]: Filter is dublicated here

**Comment by brianteeman** [comment]
@zero-24 fixed that

**Comment by Quy** [comment]
In profile.xml, add a line before lines 274 and 295.

**Comment by brianteeman** [comment]
@quy done

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on f07e24035a599dfd1ce72cc4bd60c6325b536d5c<br /><br />Code review.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15932">issues.joomla.org/tracker/joomla-cms/15932</a>.</sub>

**Comment by brianteeman** [comment]
Updated branch to restart travis - no need for new tests

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on ddcaf07b54bdd49d8a1e1a31b2f79debc8923a9d<br /><br />ok on code review.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15932">issues.joomla.org/tracker/joomla-cms/15932</a>.</sub>

**Comment by brianteeman** [comment]
RTC please?

**Comment by franz-wohlkoenig** [comment]
RTC

**Comment by brianteeman** [comment]
@wilsonge you will probably want to merge this and #15922 before you do your 3.8->4 merge

**Comment by Quy** [comment]
@brianteeman The following have blank lines to delete. I attempted to submit patches but they contain previous commits. Can you remove them or tell me how to do via Github interface?  Thanks.

plugins/authentication/cookie/cookie.xml
plugins/authentication/ldap/ldap.xml
plugins/content/emailcloak/emailcloak.xml
plugins/editors/tinymce/tinymce.xml
plugins/fields/textarea/params/textarea.xml
plugins/fields/textarea/textarea.xml

**Comment by zero-24** [comment]
@quy do you have Glip? If yes please contace me and i can help you with github. If you do not have a account please send me a mail to tobias.zulauf[at]community.joomla.org

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**